src/tests/components/CalcDisplay.test.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import CalcDisplay from '../../components/CalcDisplay'; // eslint-disable-line
```

Comment:
Currently, this test is still failing because you have used the wrong file name.

## Solution:-

Please consider reviewing the import statement below for a better understanding and you can use this path to make the test work.

```
import CalcDisplay from '../../components/calcDisplay.jsx'; 
```